src/components/dashboard/container/SpecificConcept.jsx

```diff
@@ -101,7 +102,7 @@ export class SpecificConcept extends Component {
         <div className="container-fluid pt-3">
           <div className="row justify-content-center">
             <div className="col-10 offset-sm-1">
-              <Link to={`/concepts/${userType}/${typeName}/${dictionaryId}/${dictionaryName}`} id="dictionary-link">
+              <Link to={`/concepts/${userType}/${typeName}/${dictionaryId}/${dictionaryName}/${language}`} id="dictionary-link">
```

Comment:
For us to take care of further changes in the URL, you could just store the pathname because its dynamic. Also, I don't think this solves the whole problem, for example when you view concepts in a source this link will still throw an error.

Reply:
Thank you for the feedback.
I have made the relevant changes